### ISSUE TITLE
Questions about usage

### ISSUE THREAD

**Issue by wanghongN** [Hengzhe-Zhang/PS-Tree#9]
Hello, sorry to  border you.

I have a question about how to use the pstree.

I have downloaded the source and use "python setup.py install" to install the package. When I try to use PSTreeRegressor as your code shows, I found that I have no idea about how to import PSTreeRegressor. Is "import pstree as pt" and "pt.PSTreeRegressor(......)" correct?

I used it in this way, but it showed that module 'pstree' has no attribute 'PSTreeRegressor'.

It would be appreciated if  you could give me some complete example of how to use it.

Thanks a lot!

**Comment by hengzhe-zhang** [comment]
https://github.com/hengzhe-zhang/PS-Tree/blob/9461fbdae74a34e8802e6868897730c1acb10399/pstree/cluster_gp_sklearn.py#L1101

**Comment by wanghongN** [comment]
> [PS-Tree/pstree/cluster_gp_sklearn.py](https://github.com/hengzhe-zhang/PS-Tree/blob/9461fbdae74a34e8802e6868897730c1acb10399/pstree/cluster_gp_sklearn.py#L1101)
> 
> Line 1101 in [9461fbd](/hengzhe-zhang/PS-Tree/commit/9461fbdae74a34e8802e6868897730c1acb10399)
> 
>  class PSTreeRegressor(NormalizationRegressor):

**Comment by wanghongN** [comment]
Does this package only suitable for Linux platform? I try to use it in Windows, but the "glmnet" package could not be well installed.

**Comment by hengzhe-zhang** [comment]
> Does this package only suitable for Linux platform? I try to use it in Windows, but the "glmnet" package could not be well installed.

Yes, the package is currently only suitable for Linux systems. I’m not sure when glmnet will support Windows. Once glmnet supports Windows, the package should be able to run on Windows smoothly as well.
